enh-ruby-mode.el

```diff
@@ -751,7 +751,7 @@ If the result is do-end block, it will always be multiline."
             (enh-ruby-brace-to-do-end block-start block-end)
           (enh-ruby-do-end-to-brace block-start block-end)))))
 
-(defun enh-ruby-imenu-create-index-in-block (prefix beg end)
+(defun enh-ruby-imenu-create-index-in-block (beg end)
```

Comment:
not sure about this change... It is essentially an API change. Prolly no impact, but I need to poke around